pkg/model/project.go

```diff
@@ -351,51 +351,27 @@ func (p *ProjectSSOConfig_GitHub) GenerateAuthCodeURL(project, callbackURL, stat
 	return authURL, nil
 }
 
-// FilterRBACRoles filter rbac roles for built-in or not.
-func (p *Project) FilterRBACRoles(isBuiltin bool) []*ProjectRBACRole {
-	v := make([]*ProjectRBACRole, 0, len(p.RbacRoles))
-	for _, role := range p.RbacRoles {
-		if role.IsBuiltin != isBuiltin {
-			continue
+// IsRBACRoleExists checks whether the RBAC role is exists.
+func (p *Project) IsRBACRoleExists(name string) bool {
```

Comment:
nits, `HasRBACRole(name string)` is more readable, since the role belongs to the project, and the project should know it has a given role name or not 😄